MIGRATING.MD

```diff
@@ -64,32 +79,37 @@ OpenTelemetry Java Instrumentation.
 ### Configuration setting changes
 
 These SignalFx Java Agent system properties correspond to the following
-OpenTelemetry system properties:
+OpenTelemetry system properties (NOTE: for exporter - specific properties we use `jaeger` as an example):
 
 | SignalFx system property | OpenTelemetry system property |
 | ------------------------ | ----------------------------- |
-| `signalfx.service.name` | `otel.resource.attributes=service.name=my-java-app` |
+| `signalfx.service.name` | `otel.resource.attributes=service.name=<name of the service>` |
+| `signalfx.env` | `otel.resource.attributes=deployment.environment=<name of the environment>` |
 | `signalfx.endpoint.url` | `otel.exporter.jaeger.endpoint` |
-| `signalfx.tracing.enabled` | `otel.trace.enabled` |
-| `signalfx.span.tags` | `otel.resource.attributes` |
-| `signalfx.recorded.value.max.length` | `otel.config.max.attr.length` |
-| `signalfx.db.statement.max.length` | `otel.config.max.attr.length` | 
+| `signalfx.tracing.enabled` | `otel.javaagent.enabled` |
+| `signalfx.integration.<name>.enabled=false` | `otel.instrumentation.[id].enabled=false` | 
+| `signalfx.span.tags` | `otel.resource.attributes=<comma separated key=value pairs>` |
 | `signalfx.trace.annotated.method.blacklist` | `otel.trace.annotated.methods.exclude` |
 | `signalfx.trace.methods` | `otel.trace.methods` |
-| `signalfx.integration.<name>.enabled=true` | `otel.integration.[id].enabled=false` | 
+| `signalfx.server.timing.context` | `splunk.context.server-timing.enabled` |
+
+
+Additional info about disabling a particular instrumentations can be found in the [OpenTelemetry Java Instrumentation docs](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/suppressing-instrumentation.md)
 
 These SignalFx Java Agent environment variables correspond to the following
 OpenTelemetry environment variables:
 
 | SignalFx environment variable | OpenTelemetry environment variable |
 | ----------------------------- | ---------------------------------- |
 | `SIGNALFX_SERVICE_NAME` | `OTEL_RESOURCE_ATTRIBUTES=service.name=my-java-app` |
+| `SIGNALFX_ENV` | `OTEL_RESOURCE_ATTRIBUTES=deployment.environment=<name of the environment>` |
 | `SIGNALFX_ENDPOINT_URL` |`OTEL_EXPORTER_JAEGER_ENDPOINT` |
-| `SIGNALFX_TRACING_ENABLED` | `OTEL_TRACE_ENABLED` |
+| `SIGNALFX_TRACING_ENABLED` | `OTEL_JAVAAGENT_ENABLED` |
+| `SIGNALFX_INTEGRATION_<name>_ENABLED=false` | `OTEL_INSTRUMENTATION_[id]_ENABLED=false` |
```

Comment:
```suggestion
| `SIGNALFX_INTEGRATION_<name>_ENABLED=false` | `OTEL_INSTRUMENTATION_<name>_ENABLED=false` |
```

Reply:
Due to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/suppressing-instrumentation.md I'll stay with ID.